client/dashboard/task-list/style.scss

```diff
@@ -194,3 +194,25 @@
 		font-size: 16px;
 	}
 }
+
+.woocommerce-task-payments {
+	.woocommerce-list__item .woocommerce-list__item-after {
+		align-self: start;
+		margin-left: $gap;
+		margin-top: $gap-large;
+	}
+
+	.woocommerce-task-payments__woocommerce-services-options {
+		border-top: 1px solid $new-muriel-gray-5;
```

Comment:
I'm also seeing the same border top for `woocommerce-list__item-text` in the designs and the payment borders not extending across the checkboxes.

<img width="528" alt="Screen Shot 2019-08-29 at 7 41 04 PM" src="https://user-images.githubusercontent.com/10561050/63937421-f66fbf80-ca94-11e9-8e90-708152db9bf3.png">